src/components/Book.js

```diff
@@ -1,21 +1,120 @@
-import React from 'react';
-// eslint-disable-next-line import/no-extraneous-dependencies
-import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+import Form from './Form';
 
-const Book = ({ title, author }) => (
-  <>
-    <div>
-      <h3>{title}</h3>
-      <p>{author}</p>
+function Book() {
+  const bookstore = useSelector((state) => state.books.bookstore);
+  const categories = useSelector((state) => state.categories.categories);
+  const [selected, setSelected] = useState();
+  const dispatch = useDispatch();
+  const selectHandler = (e) => {
+    setSelected(e.target.value);
+    // console.log(bookstore);
+  };
```

Comment:
- [ ] Please try and remove the commented code as raised earlier, this will make your codebase clean according to the best practises. :info